pygazebo/pygazebo.cc

```diff
@@ -201,7 +201,7 @@ class Agent : public Model {
 
       gazebo::sensors::ContactSensorPtr sensor =
           std::dynamic_pointer_cast<gazebo::sensors::ContactSensor>(
-              mgr->GetSensor(contact_sensor_name));
```

Comment:
Pls remove "gazebo::sensors::SensorManager* mgr = gazebo::sensors::SensorManager::Instance()" before this line.

Reply:
It is removed :)